docs/aws/audit/redshiftmonitoring/rules/emr_kerberos_enabled_remediation.mdx

```diff
@@ -1,6 +1,258 @@
 
 ### Triage and Remediation
 <Tabs>
+
+
+<Tab title='Prevention'>
+### How to Prevent
+<AccordionGroup>
+<Accordion title='Using Console' defaultOpen='true'>
+To prevent Amazon EMR clusters from being created without Kerberos enabled using the AWS Management Console, follow these steps:
+
+1. **Navigate to the EMR Console:**
+   - Sign in to the AWS Management Console.
+   - Open the Amazon EMR console at [https://console.aws.amazon.com/elasticmapreduce/](https://console.aws.amazon.com/elasticmapreduce/).
+
+2. **Create or Edit a Security Configuration:**
+   - In the left-hand navigation pane, choose **Security configurations**.
+   - Click on **Create security configuration** or select an existing security configuration to edit.
+   - Under **Kerberos authentication**, check the box to enable Kerberos authentication.
+   - Provide the necessary details for the Kerberos configuration, such as the KDC (Key Distribution Center) and realm information.
+
+3. **Apply the Security Configuration to Your EMR Cluster:**
+   - When creating a new EMR cluster, in the **Security and access** section, select the security configuration that has Kerberos enabled.
+   - If editing an existing cluster, ensure that the cluster is terminated and then create a new cluster with the appropriate security configuration.
+
+4. **Review and Launch:**
+   - Review all the settings to ensure Kerberos is enabled.
+   - Click **Create cluster** to launch the EMR cluster with the Kerberos-enabled security configuration.
+
+By following these steps, you can ensure that your Amazon EMR clusters have Kerberos enabled, enhancing the security of your data and operations.
+</Accordion>
+
+<Accordion title='Using CLI'>
+To prevent Amazon EMR clusters from being created without Kerberos enabled using AWS CLI, you can follow these steps:
+
+1. **Create a Security Configuration with Kerberos Enabled:**
+   First, create a security configuration that enables Kerberos. This configuration will be used when creating EMR clusters.
+
+   ```sh
+   aws emr create-security-configuration --name MyKerberosConfig --security-configuration '{
+     "AuthenticationConfiguration": {
+       "KerberosConfiguration": {
+         "Provider": "ClusterDedicatedKdc",
+         "ClusterDedicatedKdcConfiguration": {
+           "TicketLifetimeInHours": 24,
+           "CrossRealmTrustPrincipalPassword": "YourPassword"
```

Comment:
<b>Secret:</b> Secret Keyword <br/>
                    <b><a href="https://console.cloudanix.com/code?t=secrets" target='_blank'>Click here for more details</a></b>